src/main/java/com/spotify/docker/client/messages/NetworkSettings.java

```diff
@@ -70,6 +71,42 @@
   @JsonProperty("Networks")
   public abstract ImmutableMap<String, AttachedNetwork> networks();
 
+  @Nullable
+  @JsonProperty("EndpointID")
+  public abstract String endpointId();
+
+  @Nullable
+  @JsonProperty("SandboxID")
+  public abstract String sandboxId();
+
+  @Nullable
+  @JsonProperty("SandboxKey")
+  public abstract String sandboxKey();
+
+  @Nullable
+  @JsonProperty("HairpinMode")
+  public abstract Boolean hairpinMode();
+
+  @Nullable
+  @JsonProperty("LinkLocalIPv6Address")
+  public abstract String linkLocalIpv6Address();
```

Comment:
One last nitpick. Do you mind changing these method and the one below to have `IP` instead of `Ip` to be consistent with the two other methods below that and other classes' methods?